challenges/fifoAnimalShelter/fifo-animal-shelter.js

```diff
@@ -0,0 +1,39 @@
+const Queue = require('../StacksAndQueues/stacks-and-queues');
+
+class Animal {
+  constructor() {
+    this.dog = new Queue.Queue();
+    this.cat = new Queue.Queue();
+  }
+
+  enqueue(animal) {
+    if(animal.type === 'dog'){
```

Comment:
instead of having two seperate if statements that lead to the same line of code, you could put an "or" operator ( || ) in a single if statement 

if(animal.type === 'dog' || animal.type === 'cat')